google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudConfigurable.java

```diff
@@ -35,17 +39,40 @@
   private static final String MORE_INFO_URI_OPEN_TAG =
       "<a href='https://cloud.google.com/appengine/'>";
   private static final String MORE_INFO_URI_CLOSE_TAG = "</a>";
+  private static final String PSEUDO_GOOGLE_SDK_LINK = "http://google-sdk/";
 
   private String displayName = GctBundle.message("appengine.name");
   private JPanel mainPanel;
   private JTextPane appEngineMoreInfoLabel;
-  private CloudSdkPanel cloudSdkPanel;
 
   /** Initialize the UI. */
-  public AppEngineCloudConfigurable() {
-    appEngineMoreInfoLabel.setText(
+  AppEngineCloudConfigurable() {
+    // in case managed SDK is not available yet, provide direct link to SDK settings to allow
+    // familiar SDK setup via App Engine Server panel.
+    StringBuilder messageBuilder = new StringBuilder();
+    if (!ServiceManager.getService(PluginInfoService.class).shouldEnable(GctFeature.MANAGED_SDK)) {
+      messageBuilder.append(
+          GctBundle.message("appengine.cloud.sdk.settings", PSEUDO_GOOGLE_SDK_LINK));
+      System.out.println(messageBuilder.toString());
+      messageBuilder.append("<p/>");
+    }
+    messageBuilder.append(
         GctBundle.message("appengine.more.info", MORE_INFO_URI_OPEN_TAG, MORE_INFO_URI_CLOSE_TAG));
-    appEngineMoreInfoLabel.addHyperlinkListener(new BrowserOpeningHyperLinkListener());
+
+    appEngineMoreInfoLabel.setText(messageBuilder.toString());
+    appEngineMoreInfoLabel.addHyperlinkListener(
```

Comment:
Wondering if it is cleaner to overload the `appEngineMoreInfoLabel` for 2 use-cases like this instead of creating a new label used just for the cloud sdk link? I'm open to it, but just want to put that option out there.

Reply:
Oh you mean have the separate text with placeholders for feature based scenario vs. standard scenario? It looks like we use the same label here, can you please clarify if I missed your point a bit.